examples/tree_1d_dgsem/elixir_shallowwater_parabolic_bowl.jl

```diff
@@ -0,0 +1,109 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations1D(gravity_constant=9.81)
+cfl = 0.3
+
+
+function parabolic_bowl_analytic_1D_H(gravity, x,t)
+  a = 1
+  h_0 = 0.1
+  σ = 0.5
+  ω = sqrt(2*gravity*h_0)/a
+
+  H = σ * h_0/a^2 * (2*x[1]*cos(ω*t) - σ) + h_0
+  return H
+end
+
+# Implemented based on Wintermeyer (here seen in 2D)
```

Comment:
You should add a docstring with a reference and DOI to the Wintermeyer paper where this initial condition came from. Here you can also mention that an analytical solution exists which is also in that paper.